modules/custom/az_finder/src/Plugin/better_exposed_filters/filter/AzFinderTaxonomyIndexTidWidget.php

```diff
@@ -0,0 +1,413 @@
+use Drupal\Core\Template\Attribute;
+use Drupal\taxonomy\Plugin\views\filter\TaxonomyIndexTid;
+use Drupal\views\ViewExecutable;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Finder widget implementation.
+ *
+ * @BetterExposedFiltersFilterWidget(
+ *   id = "az_finder_tid_widget",
+ *   label = @Translation("Quickstart Finder Term ID Widget"),
+ * )
+ */
+class AZFinderTaxonomyIndexTidWidget extends FilterWidgetBase implements ContainerFactoryPluginInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * The configuration for the plugin.
+   *
+   * @var array
+   */
+  protected $configuration;
+
+  /**
+   * The renderer service.
+   *
+   * @var \Drupal\Core\Render\RendererInterface
+   */
+  protected $renderer;
+
+  /**
+   * The entity type manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The AZFinderIcons service.
+   *
+   * @var \Drupal\az_finder\AZFinderIcons
+   */
+  protected $AZFinderIcons;
+
+  /**
+   * Constructs a new AzFinderWidget object.
+   *
+   * @param array $configuration
+   *   The plugin configuration.
+   * @param string $plugin_id
+   *   The plugin ID.
+   * @param mixed $plugin_definition
+   *   The plugin definition.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   The renderer service.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager service.
+   * @param \Drupal\az_finder\AZFinderIcons $az_finder_icons
+   *   The AZFinderIcons service.
+   */
+  public function __construct(
+    array $configuration,
+    $plugin_id,
+    $plugin_definition,
+    RendererInterface $renderer,
+    EntityTypeManagerInterface $entity_type_manager,
+    AZFinderIcons $az_finder_icons
+  ) {
+    parent::__construct(
+      $configuration,
+      $plugin_id,
+      $plugin_definition
+    );
+    $this->configuration = $configuration;
+    $this->renderer = $renderer;
+    $this->entityTypeManager = $entity_type_manager;
+    $this->AZFinderIcons = $az_finder_icons;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(
+    ContainerInterface $container,
+    array $configuration,
+    $plugin_id,
+    $plugin_definition
+  ) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('renderer'),
+      $container->get('entity_type.manager'),
+      $container->get('az_finder.icons')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return parent::defaultConfiguration() + [
+      'advanced' => [
+        'collapsible' => FALSE,
+        'is_secondary' => FALSE,
+        'placeholder_text' => '',
+        'rewrite' => [
+          'filter_rewrite_values' => '',
+        ],
+        'sort_options' => FALSE,
+      ],
+    ];
+  }
```

Comment:
```suggestion
  public function defaultConfiguration() {
   $parent_config = parent::defaultConfiguration();
   return $parent_config;
  }
```

Reply:
I think that would work.  I also think we could just omit this method since it's not defined as an abstract method in the parent class.